main.js

```diff
@@ -0,0 +1,420 @@
+const startButton = document.getElementById('start-btn');
```

Comment:
overall, I'm able to understand your structure for the game pretty well and it makes sense in my mind. I think splitting it into multiple files can probably help to clean up a little bit (e.g. have a separate file `quiz.js` and `results.js` .. )